operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml

```diff
@@ -286,10 +286,10 @@ spec:
 
     - Tekton Pipelines: v0.56.1
```

Comment:
can we please update pipelines too here?

Reply:
updated